steely/plugins/linden.py

```diff
@@ -224,15 +232,21 @@ def give_cmd(bot, message_parts, author_id, thread_id, thread_type):
 
 
 def table_cmd(bot, message_parts, author_id, thread_id, thread_type):
+    users = list(USERDB.all())
+    if not users:
+        bot.sendMessage(
+            "No users",
```

Comment:
probably a good thing to add this check, but could you make the error message a bit more descriptive?